README.md

```diff
@@ -100,7 +100,7 @@ sudo ig daemon --host=tcp://0.0.0.0:1234
 
 Download the `gadgetctl` tools for MacOS
 ([amd64](https://github.com/inspektor-gadget/inspektor-gadget/releases/download/v0.30.0/gadgetctl-darwin-amd64-v0.30.0.tar.gz),
-[arm64](https://github.com/inspektor-gadget/inspektor-gadget/releases/download/v0.30.0/gadgetctl-darwin-arm64-v0.30.0.tar.gz)) or [windows](https://github.com/inspektor-gadget/inspektor-gadget/releases/latest/download/gadgetctl-windows-amd64-v0.30.0.tar.gz) and run the Gadget specifying the IP address of the Linux machine:
+[arm64](https://github.com/inspektor-gadget/inspektor-gadget/releases/download/v0.30.0/gadgetctl-darwin-arm64-v0.30.0.tar.gz)) or [windows](https://github.com/inspektor-gadget/inspektor-gadget/releases/download/v0.30.0/gadgetctl-windows-amd64-v0.30.0.tar.gz) and run the Gadget specifying the IP address of the Linux machine:
```

Comment:
The only way I see is to provide the commands to do it manually as we do for https://github.com/inspektor-gadget/inspektor-gadget?tab=readme-ov-file#install-locally. It would be easy for MacOS, but what about Windows? Should we provide powershell commands? 😅

Reply:
I updated the post-release PR for this:
https://github.com/inspektor-gadget/inspektor-gadget/pull/3336